macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -55,7 +55,7 @@
     'calculate_failed_count': true,
     'tests_use_temp_tables': false,
     'collect_metrics': true,
-    'upload_dbt_columns': false,
+    'upload_information_schema_columns': false,
```

Comment:
Breaking change.

Reply:
Yeah I think it's fine, given that it's off by default + only relevant for BQ